README.md
# Randomize-Duties

Project to randomize closing and opening duties for employees

# ChangeLog

Current version does not actually take employees markes as present to filter list. 


changelog.txt
Current version does not actually take employees markes as present to filter list. 

index.php
<!DOCTYTPE html>
<html>

<head>
    <title>Daily Assignments</title>
    <style>
        body {
                color:white;margin:0;background-color:#FAF0FF;
                background-image: url('http://www.digitalplatforms.co.za/wp-content/uploads/2015/11/Website-Design-Background.png');
                background-size:cover;
                 }
    </style>
    <form action='index.php' method='post'>
</head>

<body>

    <!--Header-->


    <div style='width:100%;text-align:center'>
        <h1 style='box-shadow:inset 1px 1px rgba(255,255,255,0.1), 1px 1px rgba(0,0,0,0.1);
               border-style: solid;border-color: black;border-width:.5px;
               padding:5px;WIDTH:100%;background-color:#08c;color:white;text-align:center;'>
            Daily Assignments
        </h1>
    </div>

    <!--Instructions-->


    <h3 style='text-align:center'>Select the employees to assign tasks to.</h3>

    <!--checkmarks-->


    <div style='text-align:center;margin-top:-5px;'>
        <?php
        $employees  = array("John","Ray","West","Garrett",);
        foreach($employees as $present) {
            echo "<input type='checkbox' name='$present' value='$present' style='width:4%;text-align:center;margin:auto;'>$present" ;
        };
        ?>
    </div>

    <!--Button-->


    <div style='margin:auto;text-align:center;'>
        <br><input type='submit' value='Generate' style='margin:auto'><br><br>
    </div>

    <!--Table-->


    <table style='height:43%;border-style:solid;width:90%;text-align:center;background-color:gray;margin:auto;'>
        <tr style='border-style:solid;'>
            <th style='color:red;background-color:white;border-style:solid;border-color:black;'>
                Task
            </th>
            <th style='color:green;background-color:white;border-style:solid;border-color:black;'>
                Employee
            </th>
        </tr>

        <?php
        ####Variables#######
        
        
        $employees  = array("John","Ray","West","Garrett",);
        $task = array("Trash","Floors","Vacuum","WorkBenches","Display Cases","Windows");
        $count =  0;
        
        #############tasks#################
        
        
        foreach($task as $task) {
            echo  "<tr>
            <td style='background-color:white;border-style:solid;color:black'>
            $task
            </td>" . "<td style='background-color:white;border-style:solid;color:black'>" . $employees[array_rand($employees)] . "</td>
            </tr> " ;
        }
                echo "</table>";
                #############footer info##########
                include('../Frontend/footer.php');
        ?>
</body>

</html>

footer.php
<?php
#############footer info##########

echo "<style> a {color:white;}</style>";

$footer=array("'http://start.mobilerepairtechs.com'>POS",
              "'http://www.cellphonerepair.com/gainesville-fl'>Website Front End" ,
              "'https://www.cellphonerepair.com/gainesville-fl/login/' >Website Back End",
              "'http://facebook.com/mobilerepairtechs'>Facebook",
              "'./tasks.php'>Tasks",
              "'http://mc.mobilerepairtechs.com'>Marketing Cards",
              "'http://code.raymercedes.com/clist.php'>Craigslist Ad's"

             );

sort($footer);


echo "<div style='height:20px;color:white;
      background-color:#222;bottom:0;text-align:center;
      width:100%;position:absolute;>";

echo "
        <table style='width:100%';color:white;>
           <tr style='align-text:center;width:100%;color:white;'>";


echo "<div style='color:white';>";


echo "| ";
foreach($footer as $footer) {
                                echo "<td style='text-decoration-color:white;'> <a target='_blank' href=$footer </a> | </td>";   
                            }
echo "</div>";

echo '</div>';
?>

tasks.php
<DOCTYTPE html></DOCTYTPE>
<html>

<head>
    <title>Daily Assignments</title>
    <style>
        body {
                color:white;margin:0;background-color:#FAF0FF;
                background-image: url('http://www.digitalplatforms.co.za/wp-content/uploads/2015/11/Website-Design-Background.png');
                background-size:cover;
                 }
    </style>
    <form action='tasks.php' method='post'>
</head>

<body>
    
    !--Header-->

    <div style='width:100%;text-align:center'>
        <h1 style='box-shadow:inset 1px 1px rgba(255,255,255,0.1), 1px 1px rgba(0,0,0,0.1);
               border-style: solid;border-color: black;border-width:.5px;
               padding:5px;WIDTH:100%;background-color:#08c;color:white;text-align:center;'>
            Daily Assignments
        </h1>
    </div>
    <!--Instructions-->
    
    <h3 style='text-align:center'>Select the employees to assign tasks to.</h3>
    <!--checkmarks-->
    <div style='text-align:center;margin-top:-5px;'>
        <?php
        $employees  = array("John","Ray","West","Garrett",);
        foreach($employees as $present) {
            echo "<input type='checkbox' name='$present' value='$present' style='width:4%;text-align:center;margin:auto;'>$present" ;
        };
        ?>
    </div>
    
    <!--Button-->
    
    <div style='margin:auto;text-align:center;'>
        <br><input type='submit' value='Generate' style='margin:auto'><br><br>
    </div>
    
    <!--Table-->
    
    <table style='height:43%;border-style:solid;width:90%;text-align:center;background-color:gray;margin:auto;'>
        <tr style='border-style:solid;'>
            <th style='color:red;background-color:white;border-style:solid;border-color:black;'>
                Task
            </th>
            <th style='color:green;background-color:white;border-style:solid;border-color:black;'>
                Employee
            </th>
        </tr>
    
    <?php
        ####Variables#######
        $employees  = array("John","Ray","West","Garrett",);
        $task = array("Trash","Floors","Vacuum","WorkBenches","Display Cases","Windows");
        $count =  0;
        
        #############tasks#################
        foreach($task as $task) {
            echo  "<tr>
            <td style='background-color:white;border-style:solid;color:black'>
            $task
            </td>" . "<td style='background-color:white;border-style:solid;color:black'>" . $employees[array_rand($employees)] . "</td> 
                     </tr> " ;
                   



            #  foreach($employees as $employees){
             #       echo $employees . " " . $count;
             #   $count ++; 
             # }   
}
                echo "</table>";
                #############footer info##########
                include('footer.php');
                ?>
</body>

</html>
